### PR TITLE
[I18N] default_warehouse_from_sale_team: Update translation terms

### PR DESCRIPTION
**Pull request by CarmenMiranda**
### [T#40512](https://www.vauxoo.com/web#id=40512&view_type=form&model=project.task)

By merging the two Spanish language translation files we are solving the issue of duplicated terms on the translations of this
module. And increasing the module version for this major change.

### CODE REVIEW

**Comment by CarmenMiranda** [comment]
Hi, @luisg123v Could you review this? Please

@JulioSerna I did this MR because I was having the error of duplicated terms when I was installing it on ABSA

**Review by luisg123v — CHANGES_REQUESTED** [review]
Could you reword to be as follows, please?

`[I18N] ...: Update translation terms`

**Comment by CarmenMiranda** [comment]
> Could you reword to be as follows, please?
> 
> `[I18N] ...: Update translation terms`

Done :smile:

File: default_warehouse_from_sale_team/i18n/es.po [hunk 1]
```diff
@@ -86,51 +149,88 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
 msgid "Not matter sales team will leave the user to see all the stock picking types"
 msgstr "No importa los equipos de ventas le permitirá al usuario ver todos los tipos de albarán"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,comment:default_warehouse_from_sale_team.group_manager_default_journal
+msgid "Not matter sales teams will leave the user to see all the Journal"
+msgstr "No importa el equipo de ventas le permitirá ver todos los diarios"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: Permite al usuario poder ver todos los diarios, sin importar su equipo de ventas

File: default_warehouse_from_sale_team/i18n/es.po [hunk 2]
```diff
@@ -86,51 +149,88 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
...
 #: model:res.groups,comment:default_warehouse_from_sale_team.group_manager_default_warehouse_sp
 msgid "Not matter sales teams will leave the user to see all the stock pickings"
 msgstr "No importa los equipos de ventas le permitirá al usuario ver todos los albaranes"
 
 #. module: default_warehouse_from_sale_team
-#: model:ir.model,name:default_warehouse_from_sale_team.model_stock_picking
-msgid "Picking List"
-msgstr "Albarán"
+#: model:ir.model,name:default_warehouse_from_sale_team.model_stock_picking_type
+msgid "Picking Type"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po:155]: It should be "operation type", and translated as "typo de operación"

File: default_warehouse_from_sale_team/i18n/es.po [hunk 3]
```diff
@@ -86,51 +149,88 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
...
+msgid "Sale Order"
 msgstr "Pedido de venta"
 
 #. module: default_warehouse_from_sale_team
-#: model:ir.model,name:default_warehouse_from_sale_team.model_crm_case_section
-msgid "Sales Teams"
-msgstr "Equipos de ventas"
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_account_journal__section_id
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_ir_sequence__section_id
+msgid "Sale team"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: sale**s** team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 4]
```diff
@@ -86,51 +149,88 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
...
 
 #. module: default_warehouse_from_sale_team
-#: help:res.users,sale_team_ids:0
+#: model:ir.model,name:default_warehouse_from_sale_team.model_ir_sequence
+msgid "Sequence"
+msgstr "Secuencia"
+
+#. module: default_warehouse_from_sale_team
+#: model_terms:ir.ui.view,arch_db:default_warehouse_from_sale_team.view_crm_team_form_inherit
+msgid "Settings warehouse from sale team"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: warehouse settings for the sales team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 5]
```diff
@@ -86,51 +149,88 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
...
+msgstr "Equipos de ventas de usuarios."
 
 #. module: default_warehouse_from_sale_team
 #: model:ir.model,name:default_warehouse_from_sale_team.model_res_users
 msgid "Users"
 msgstr "Usuarios"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,comment:default_warehouse_from_sale_team.group_limited_default_warehouse_journal
+msgid "View only the Journal which warehouses match with the user sales teams default warehouses"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - View only the Journal whose warehouse match with the user's sales teams default warehouse (or similar)
- Spanish translation is also broken

File: default_warehouse_from_sale_team/i18n/es.po [hunk 6]
```diff
@@ -142,64 +242,14 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
-#: field:stock.picking,warehouse_id:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_stock_picking__warehouse_id
 msgid "Warehouse"
 msgstr "Almacén"
 
 #. module: default_warehouse_from_sale_team
-#: code:addons/default_warehouse_from_sale_team/models/res_users.py:30
+#: code:addons/default_warehouse_from_sale_team/models/res_users.py:23
 #, python-format
 msgid "You can not set %s sale team as default because the user do not belongs to the sale teams.\n"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - sale**s** team
- user **does not**

File: default_warehouse_from_sale_team/i18n/es.po [hunk 7]
```diff
@@ -142,64 +242,14 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
-#: field:crm.case.section,journal_stock_id:0
-msgid "Journal stock valuation"
-msgstr "Diario para valoración de existencias"
-
-#. module: default_warehouse_from_sale_team
-#: view:crm.case.section:default_warehouse_from_sale_team.view_crm_case_section_form_inherit
-msgid "Journal's sale team"
-msgstr "Diarios de equipo de ventas"
+"Please go to Sales > Settings > Users > Sales Teams if you will like to add this user to the sales team"
+msgstr "No puede establecer %s equipo de venta como predeterminado porque el usuario no pertenece a los equipos de venta.\n"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - establecer el equipo de ventas %s
- no pertenece al equipo de ventas

File: default_warehouse_from_sale_team/i18n/es.po [hunk 8]
```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
 msgstr "Acceso a todos los albaranes"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,create_uid:0
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_team_ids
+msgid "Choose the Journals that user with this sale team can see"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - Define the Journals an user that belongs to this sales team can see
- Also translation

File: default_warehouse_from_sale_team/i18n/es.po [hunk 9]
```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
...
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,__last_update:0
+#: model:ir.model,name:default_warehouse_from_sale_team.model_account_invoice
+msgid "Invoice"
+msgstr "Factura"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_stock_id
+msgid "It indicates journal to be used when move line is created withthe warehouse of this sale team"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - the journal to be used
- a move line
- with the
- sales team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 10]
```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
...
+
+#. module: default_warehouse_from_sale_team
+#: model_terms:ir.ui.view,arch_db:default_warehouse_from_sale_team.view_crm_team_form_inherit
+msgid "Journal's sale team"
+msgstr "Diarios de equipo de ventas"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_crm_team__journal_team_ids
+msgid "Journal's sales teams"
+msgstr "Diarios de equipos de ventas"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po:118]: This should be with the above term, whose only difference is sales team vs sale team

**Review by luisg123v — CHANGES_REQUESTED** [review]
sale team -> sales team

File: default_warehouse_from_sale_team/i18n/es.po (hunk 2 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po:155]: This Description comes from the original model in Odoo

File: default_warehouse_from_sale_team/i18n/es.po (hunk 1 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 3 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 4 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 6 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 9 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 7 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 2 above)
Comment (luisg123v) [reply default_warehouse_from_sale_team/i18n/es.po:155]: OK, then nothing to do here, resolving.

File: default_warehouse_from_sale_team/i18n/es.po [hunk 11]
```diff
@@ -142,64 +233,19 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
-#: field:crm.case.section,journal_team_ids:0
-msgid "Journal's sales teams"
-msgstr "Diarios de equipo de ventas"
-
-#. module: default_warehouse_from_sale_team
-#: view:crm.case.section:default_warehouse_from_sale_team.view_crm_case_section_form_inherit
-msgid "Journals"
-msgstr "Diarios"
+#: model_terms:ir.ui.view,arch_db:default_warehouse_from_sale_team.view_crm_team_form_inherit
+msgid "Warehouse settings for the sale team"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: sales team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 12]
```diff
@@ -142,64 +233,19 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
-msgid "Journal stock valuation"
-msgstr "Diario para valoración de existencias"
-
-#. module: default_warehouse_from_sale_team
-#: view:crm.case.section:default_warehouse_from_sale_team.view_crm_case_section_form_inherit
-msgid "Journal's sale team"
-msgstr "Diarios de equipo de ventas"
+#: code:addons/default_warehouse_from_sale_team/models/res_users.py:23
+#, python-format
+msgid "You can not set %s sales team as default because the user does not belongs to the sale teams.\n"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: You can not set the sales team %s as default

File: default_warehouse_from_sale_team/i18n/es.po [hunk 13]
```diff
@@ -142,64 +233,19 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
-#: code:addons/default_warehouse_from_sale_team/models/res_users.py:30
-#, python-format
-msgid "You can not set %s sale team as default because the user do not belongs to the sale teams.\n"
-"Please go to Sales > Sales > Sales Teams menu if you will like to add this user to the sales team"
-msgstr "No puedes agregar a %s como el equipo de ventas por defecto porque el usuario no pertenece a dicho equipo de ventas.\n"
-"Por favor vaya al menú Ventas > Ventas > Equipos de Ventas si desea agregar el usuario al equipo de ventas"
-
-#. module: default_warehouse_from_sale_team
-#: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_journal
-msgid "Access to all Journal"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po:159]: all journal**s**

File: default_warehouse_from_sale_team/i18n/es.po [hunk 14]
```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
...
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,write_date:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__write_date
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__write_date
 msgid "Last Updated on"
 msgstr "Última actualización en"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_limited_default_warehouse_journal
+msgid "Limited access to Journal (filtered by sale teams)"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: sales team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 15]
```diff
@@ -86,51 +149,79 @@ msgstr "Acceso limitado a los tipos de albarán (filtrados por equipo de venta)"
 msgid "Not matter sales team will leave the user to see all the stock picking types"
 msgstr "No importa los equipos de ventas le permitirá al usuario ver todos los tipos de albarán"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,comment:default_warehouse_from_sale_team.group_manager_default_journal
+msgid "Not matter sales teams will leave the user to see all the Journal"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: allow the user to see all journals, regardless of sales teams

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: default_warehouse_from_sale_team/i18n/es.po (hunk 8 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 10 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po:118]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 11 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 12 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 13 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po:159]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 14 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 15 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 5 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po [hunk 16]
```diff
@@ -26,59 +31,117 @@ msgid "Access to all pickings"
...
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_limited_default_warehouse_journal
+msgid "Limited access to Journal (filtered by sales team)"
+msgstr "Acceso limitado a diarios(Filtrados por equipo de ventas)"
+
 #. module: default_warehouse_from_sale_team
 #: model:res.groups,name:default_warehouse_from_sale_team.group_limited_default_warehouse_sp
-msgid "Limited access to pickings (filtered by sale teams)"
+msgid "Limited access to pickings (filtered by sales team)"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - pickings -> transfers
- Albaranes -> transferencias

File: default_warehouse_from_sale_team/i18n/es.po [hunk 17]
```diff
@@ -26,59 +31,117 @@ msgid "Access to all pickings"
...
 msgstr "Acceso limitado a los albaranes (filtrados por equipo de venta)"
 
 #. module: default_warehouse_from_sale_team
 #: model:res.groups,name:default_warehouse_from_sale_team.group_limited_default_warehouse_spt
-msgid "Limited access to pickings types (filtered by sale teams)"
+msgid "Limited access to pickings types (filtered by sales team)"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - operation types
- Tipos de operación

File: default_warehouse_from_sale_team/i18n/es.po [hunk 18]
```diff
@@ -92,45 +155,73 @@ msgid "Not matter sales teams will leave the user to see all the stock pickings"
 msgstr "No importa los equipos de ventas le permitirá al usuario ver todos los albaranes"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - albaranes -> transferencias
- This needs a similar fix as the term for journals (Allow the user to see all transfers, regardless of sales teams)

File: default_warehouse_from_sale_team/i18n/es.po [hunk 19]
```diff
@@ -142,64 +233,19 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
-msgid "Journal stock valuation"
-msgstr "Diario para valoración de existencias"
-
-#. module: default_warehouse_from_sale_team
-#: view:crm.case.section:default_warehouse_from_sale_team.view_crm_case_section_form_inherit
-msgid "Journal's sale team"
-msgstr "Diarios de equipo de ventas"
+#: code:addons/default_warehouse_from_sale_team/models/res_users.py:23
+#, python-format
+msgid "You can not set the sales team %s as default because the user does not belongs to the sale teams.\n"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: belongs to **that** sale**s** team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 20]
```diff
@@ -142,64 +233,19 @@ msgid "View only the stock pickings which warehouses match with the user sales t
...
+"Please go to Sales > Settings > Users > Sales Teams if you will like to add this user to the sales team"
+msgstr "No puede establecer el equipo de ventas %s como predeterminado porque el usuario no pertenece al equipo de ventas.\n"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: no pertenece a **dicho** equipo de ventas (that same word was on the original term)

File: default_warehouse_from_sale_team/security/res_groups.xml [hunk 21]
```diff
@@ -30,13 +30,13 @@
             </record>
 
             <record id="group_limited_default_warehouse_journal" model="res.groups">
-                <field name="name">Limited access to Journal (filtered by sale teams)</field>
-                <field name="comment">View only the Journal which warehouses match with the user sales teams default warehouses</field>
+                <field name="name">Limited access to Journal (filtered by sales team)</field>
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/security/res_groups.xml]: journal**s**

File: default_warehouse_from_sale_team/i18n/es.po [hunk 22]
```diff
@@ -26,59 +31,117 @@ msgid "Access to all pickings"
...
+msgstr "Diario"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_crm_team__journal_stock_id
+msgid "Journal stock valuation"
+msgstr "Diario para valoración de existencias"
+
+#. module: default_warehouse_from_sale_team
+#: model_terms:ir.ui.view,arch_db:default_warehouse_from_sale_team.view_crm_team_form_inherit
+msgid "Journal's sale team"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: sale**s** team

File: default_warehouse_from_sale_team/i18n/es.po [hunk 23]
```diff
@@ -26,59 +31,117 @@ msgid "Access to all pickings"
 msgstr "Acceso a todos los albaranes"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: - albaranes -> transferencias
- pickings -> transfers

File: default_warehouse_from_sale_team/i18n/es.po [hunk 24]
```diff
@@ -4,17 +4,22 @@
...
 "Plural-Forms: \n"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_journal
+msgid "Access to all Journals"
+msgstr "Acceso a todos los Diarios"
+
 #. module: default_warehouse_from_sale_team
 #: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_warehouse_spt
 msgid "Access to all picking types"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: operation types

File: default_warehouse_from_sale_team/i18n/es.po (hunk 18 above)
Comment (luisg123v) [reply default_warehouse_from_sale_team/i18n/es.po]: also picking and picking types need this same fix

File: default_warehouse_from_sale_team/i18n/es.po [hunk 25]
```diff
@@ -92,45 +155,73 @@ msgid "Not matter sales teams will leave the user to see all the stock pickings"
...
 msgstr "Usuarios"
 
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,comment:default_warehouse_from_sale_team.group_limited_default_warehouse_journal
+msgid "View only the Journal whose warehouse match the configured for the user as default warehouse"
+msgstr "Ver sólo el Diario cuyo almacén coincide con los almacenes configurados por el usuario como almacén por defecto"
+
 #. module: default_warehouse_from_sale_team
 #: model:res.groups,comment:default_warehouse_from_sale_team.group_limited_default_warehouse_spt
 msgid "View only the stock picking types which warehouses match with the user sales teams default warehouses"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es.po]: operation types

**Review by luisg123v — CHANGES_REQUESTED** [review]
There still are some:
- Picking -> transfer
- Picking type -> operation type
- Sale team -> sales team

File: default_warehouse_from_sale_team/i18n/es.po (hunk 16 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 17 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 18 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 19 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 20 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 22 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 23 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 24 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/i18n/es.po (hunk 25 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/i18n/es.po]: Done

File: default_warehouse_from_sale_team/security/res_groups.xml (hunk 21 above)
Comment (CarmenMiranda) [reply default_warehouse_from_sale_team/security/res_groups.xml]: Done

**Review by luisg123v — APPROVED** [review]
outstanding work!

**Pull request merged**